.cproject

```diff
@@ -59,39 +59,24 @@
-									<listOptionValue builtIn="false" value="&quot;/${ProjName}/src/application/health_checks&quot;"/>
-									<listOptionValue builtIn="false" value="&quot;/${ProjName}/src/application/imu_handler&quot;"/>
-									<listOptionValue builtIn="false" value="&quot;/${ProjName}/src/application/logger&quot;"/>
-									<listOptionValue builtIn="false" value="&quot;/${ProjName}/src/application/state_estimation&quot;"/>
-									<listOptionValue builtIn="false" value="&quot;/${ProjName}/src/drivers/adc&quot;"/>
-									<listOptionValue builtIn="false" value="&quot;/${ProjName}/src/drivers/i2c&quot;"/>
-									<listOptionValue builtIn="false" value="&quot;/${ProjName}/src/drivers/led&quot;"/>
-									<listOptionValue builtIn="false" value="&quot;/${ProjName}/src/drivers/lsm6dsv32x&quot;"/>
-									<listOptionValue builtIn="false" value="&quot;/${ProjName}/src/drivers/sd_card&quot;"/>
-									<listOptionValue builtIn="false" value="&quot;/${ProjName}/src/drivers/timer&quot;"/>
-									<listOptionValue builtIn="false" value="&quot;/${ProjName}/src/drivers/uart&quot;"/>
 									<listOptionValue builtIn="false" value="&quot;${workspace_loc:/${ProjName}/src}&quot;"/>
 									<listOptionValue builtIn="false" value="&quot;${workspace_loc:/${ProjName}/src/drivers}&quot;"/>
 									<listOptionValue builtIn="false" value="&quot;${workspace_loc:/${ProjName}/src/application}&quot;"/>
+									<listOptionValue builtIn="false" value="&quot;${workspace_loc:/processor_board/Core/Inc}&quot;"/>
+									<listOptionValue builtIn="false" value="&quot;${workspace_loc:/processor_board/Drivers/STM32H7xx_HAL_Driver/Inc}&quot;"/>
+									<listOptionValue builtIn="false" value="&quot;${workspace_loc:/processor_board/Drivers/STM32H7xx_HAL_Driver/Inc/Legacy}&quot;"/>
+									<listOptionValue builtIn="false" value="&quot;${workspace_loc:/processor_board/Drivers/CMSIS/Include}&quot;"/>
+									<listOptionValue builtIn="false" value="&quot;${workspace_loc:/processor_board/Drivers/CMSIS/Device/ST/STM32H7xx/Include}&quot;"/>
+									<listOptionValue builtIn="false" value="&quot;${workspace_loc:/processor_board/FATFS/Target}&quot;"/>
+									<listOptionValue builtIn="false" value="&quot;${workspace_loc:/processor_board/FATFS/App}&quot;"/>
+									<listOptionValue builtIn="false" value="&quot;${workspace_loc:/processor_board/Middlewares/Third_Party/FreeRTOS/Source/include}&quot;"/>
+									<listOptionValue builtIn="false" value="&quot;${workspace_loc:/processor_board/Middlewares/Third_Party/FreeRTOS/Source/CMSIS_RTOS_V2}&quot;"/>
+									<listOptionValue builtIn="false" value="&quot;${workspace_loc:/processor_board/Middlewares/Third_Party/FreeRTOS/Source/portable/GCC/ARM_CM4F}&quot;"/>
+									<listOptionValue builtIn="false" value="&quot;${workspace_loc:/processor_board/Middlewares/Third_Party/FatFs/src}&quot;"/>
+									<listOptionValue builtIn="false" value="&quot;${workspace_loc:/processor_board/Core/ThreadSafe}&quot;"/>
```

Comment:
Is there a difference between using `${ProjName}` and `processor_board`? Is this autogenerated and it doesn't even matter?

Reply:
autogenerated - will probably get overwritten next time we regen from ioc